src/main/java/dev/ebrydeu/spring_boot_library/repositrories/MessageRepository.java

```diff
@@ -0,0 +1,30 @@
+package dev.ebrydeu.spring_boot_library.repositrories;
+
+import dev.ebrydeu.spring_boot_library.domain.entities.Message;
+import org.springframework.data.repository.ListCrudRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.jpa.repository.Modifying;
+import jakarta.transaction.Transactional;
+import java.util.List;
+
+public interface MessageRepository extends ListCrudRepository<Message, Long> {
+    List<Message> findAll();
+
+    List<Message> findByAuthor(String author); // <>
+
+    List<Message> findByTitle(String title);
+
+    @Query("""
+            update Message m set m.body = ?1 where m.id = ?2
+            """)
+    @Modifying
+    @Transactional
+    int editMessageBody(String body, Long id);
+
+    @Query("""
+            update Message m set m.title = ?1 where m.id = ?2
+            """)
+    @Modifying
+    @Transactional
+    int editMessageTitle(String title, Long id);
```

Comment:
Not sure about this ones (same goes for User), you usually use `save` method both for creation and updating (both full and partial). I guess it can be here if you want, but don't now i person that do Service layer will use it at all

Reply:
I will leave if for the time been. If it will not be used, I'll delete it later.